### ISSUE TITLE
Complete or remove JSON Schema of the Core Profile

### ISSUE THREAD

**Issue by benfrancis** [w3c/wot-profile#120]
There's currently an incomplete section of the specification called [JSON Schema of the Core Profile](https://w3c.github.io/wot-profile/#core-profile-json-schema).

This section either needs to be completed, or removed, before publication.

Note that if the WoT Core Data Model section is reduced or removed as I have proposed in #10 then this section may be of limited value and the JSON schema provided by the WoT Thing Description specification would be sufficient.

**Comment by egekorkan** [comment]
I think there will be and has to more restrictions that the schema of the TD. For example, checking for `http://` in form `href` values.

**Comment by mlagally** [comment]
I completely agree with @egekorkan that we need a JSON Schema for automatic validation.
The primary goal of the Profile spec is to drive interoperability and we need additional constraints that make that happen.
The added value of the Profile is way more than just providing an unambiguous the HTTP protocol binding.

**Comment by mlagally** [comment]
tentatively assigning to @egekorkan and @FadySalama and hoping to get their help.

**Comment by egekorkan** [comment]
I agree that the contraints should be agreed on first since there is discussion on nearly every detail. After that, we can write a script that takes the TD schema creates another schema that has more constraints

**Comment by mlagally** [comment]
Defer to Profile 1.1.

**Comment by benfrancis** [comment]
Note that the Core Profile now no longer exists. JSON schemas could be created for each published profile, but that may have limited value because profiles can not be completely described by a JSON schema.

**Comment by egekorkan** [comment]
I would still see the value of having a JSON Schema for each profile, especially considering that they will be a subset of TD later on. However, a set of linting rules like with [Spectral](https://stoplight.io/open-source/spectral) might fit better since a JSON Schema will always do a strict validation (which is too assertive in some cases)
